vendor/github.com/cherryservers/cherrygo/cherrygo.go

```diff
@@ -0,0 +1,228 @@
+package cherrygo
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+)
+
+const (
+	apiURL             = "https://api.cherryservers.com/v1/"
+	cherryAuthTokenVar = "CHERRY_AUTH_TOKEN"
+	mediaType          = "application/json"
+	userAgent          = "cherry-agent-go"
+	cherryDebugVar     = "CHERRY_DEBUG"
+)
+
+// Client returns struct for client
+type Client struct {
+	client *http.Client
+	debug  bool
+
+	BaseURL *url.URL
+
+	UserAgent string
+	AuthToken string
+
+	Teams       GetTeams
+	Plans       GetPlans
+	Images      GetImages
+	Project     GetProject
+	Projects    GetProjects
+	SSHKeys     GetSSHKeys
+	SSHKey      GetSSHKey
+	Servers     GetServers
+	Server      GetServer
+	IPAddresses GetIPS
+	IPAddress   GetIP
+}
+
+// Response is the http response from api calls
+type Response struct {
+	*http.Response
+}
+
+// MakeRequest makes request to API
+func (c *Client) MakeRequest(method, path string, body, v interface{}) (*Response, error) {
+
+	url, _ := url.Parse(path)
+
+	u := c.BaseURL.ResolveReference(url)
+	fmt.Printf("\nAPI Endpoint: %v\n", u)
```

Comment:
err is there a version of cherrygo without this? Thats definitely debug logging.

Reply:
This is the state of the API. I will push a PR to cherrygo.